parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -440,7 +446,9 @@ def __init__(
                 'statistics_condition': statistics_condition,
                 'max_onboard_time': args.blueprint.max_onboard_time,
                 'onboard_task_data': self.onboard_task_data,
-                'onboarding_qualification': args.blueprint.onboarding_qualification,
+                "onboarding_qualification": args.blueprint.get(
+                    "onboarding_qualification", None
```

Comment:
nit: you don't need the `None` arg in this case, it defaults to `None` if `"onboarding_qualification"` is not an attribute